tools/do_debian_package.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+
+# Script to generate debian packages for FBInk. 
+PACKAGE="FBInk"
+AUTHOR="NiLuJe <niluje@gmail.com>"
+DESC_1="FrameBuffer eInker"
+DESC_2="a small tool to print text and images to an eInk Linux framebuffer."
+
+# Call this file from Makefile -------
+# Use $(OUT_DIR) as the first argument
+# and target arch as the second argument.
+ROOTFS_BASE="${1}"
+TARGET_ARCH="${2}"
+
+# Temp stuff.
+PKG_PATH="${ROOTFS_BASE}/pkg"
+BIN_PATH="${PKG_PATH}/usr/bin"
+DEB_PATH="${PKG_PATH}/DEBIAN"
+CONTROL="${DEB_PATH}/control"
+
+# Get version from git if posible. Fallback to a known version ;p
+# Note: version string must start with a number
+VERSION="$(git describe 2>/dev/null | sed 's/[^0-9]*//' || echo '1.9.0')"
```

Comment:
You should be able to rely on the fallback in fbink_internal.h, via something like `grep "define FBINK_VERSION" fbink_internal.h | cut -f2 -d\" | cut -f2 -dv` ;).